tests/utils.py

```diff
@@ -97,3 +100,112 @@ def configure_test_logging(argv):
 
   logging.basicConfig(level=loglevel)
   tuf.log.set_log_level(loglevel)
+
+
+class TestServerProcess():
+  """
+  <Purpose>
+    Creates a child process with the subprocess.Popen object and
+    TempFile object used for logging.
+
+   <Arguments>
+      log:
+        Logger which will be used for logging.
+
+      server:
+        Path to the server which will be runned by the subprocess.
```

Comment:
```suggestion
        Path to the server to run in the subprocess.
```
Runned is not a word, perhaps rephrase like this?